### PR TITLE
Set conditional workers views and add bio

### PR DESCRIPTION
**Pull request by ncbui**
## What type of PR is this? (check all applicable)

- [x] Feature
- [] Refactor
- [] Bug Fix
- [] Optimization
- [] Documentation Update
- [] Other (describe: )

## Description of what PR does
Adds the new bio field to worker show, view, edit. When users try to view a worker's page, they will only see worker first name, last name, and bio.

Gave the bio view a really simple UI, open to any suggestions for formatting. 

## Related PRs and/or Issues (if any)
- Completes step 3 and 4 of #52 
- Blocked by #64

## QA Instructions, Screenshots, Recordings
Log in as a worker to make sure you can see your shifts when viewing your own worker page, but only see name and bio when viewing another worker.
Screen Shot 2021-04-29 at 2 57 49 PM

## Added tests?

- [ ] yes
- [x ] no, because they aren't needed for views
- [ ] no, because I need help
- [ ] no, they will be added later (please create an Issue for it)

## Added to documentation?

- [ ] Yes, project README
- [x ] No documentation needed

### CODE REVIEW

**Review by Timilehin08 — APPROVED** [review]
Looks good! @ncbui I don't think we agreed or discussed the bio format to be in form of profile card or figures which includes avatar/picture. So i believe it's best we stick to using paragraphs for the bio. 

Welldone @ncbui

**Comment by ncbui** [comment]
Thanks Timi, have updated the bio page 
Screen Shot 2021-04-29 at 2 41 33 PM

**Comment by msarit** [comment]
Hey @ncbui, I think you also added the workers bio db change and migration to this PR. I just merged your PR that adds the workers_bio field to the database; perhaps update your fork and adjust this PR so it doesn't perform the workers_bio field add a second time? 😅

File: db/migrate/20210428203135_add_bio_to_workers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddBioToWorkers < ActiveRecord::Migration[6.0]
```
Comment (msarit) [inline db/migrate/20210428203135_add_bio_to_workers.rb:1]: Hey @ncbui, I think you also added the workers bio db change and migration to this PR. I just merged your PR that adds the workers_bio field to the database; perhaps update your fork and adjust this PR to (1) delete the duplicate migration file, so it doesn't perform the workers_bio field add a second time? 😅

**Review by msarit — CHANGES_REQUESTED** [review]

File: db/migrate/20210428203135_add_bio_to_workers.rb (hunk 1 above)
Comment (ncbui) [reply db/migrate/20210428203135_add_bio_to_workers.rb:1]: oops, sorry about that. updating PR

**Review by GabrielleEaston — APPROVED** [review]
@ncbui Great! I love that we are showing the worker profile to org!

**Review by msarit — APPROVED** [review]

**Pull request merged**